internal/controller/operator/factory/vmanomaly/models.go

```diff
@@ -0,0 +1,247 @@
+package vmanomaly
```

Comment:
I think, it's better to move all configuration related files, such as `monitoring`, `writes`, `readers` into sub-package `vmanomaly/config` or keep it at `config.go` file.

 Also, those files have non-trivial parsing logic and I was not able to fine any tests for configuration building methods, neither links for configuration source file. 

 Could you please add tests in first place? 

 And if it's possible provide links to the configuration source. It'll simplify this code for the future readers.